app/models/service.rb

```diff
@@ -97,11 +97,11 @@ def direct_vms
   end
 
   def all_vms
-    MiqPreloader.preload_and_map(subtree, :direct_vms)
+    MiqPreloader.preload_and_map(subtree, :vms)
```

Comment:
@bzwei @gmcculloug weird thing, preload_and_map comment says, it's there to avoid N+1 queries. But I saw it does N+1 queries anyway. I haven't investigated this much further.